### PR TITLE
feat(select): add placeholder (VIV-1242)

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: libs/components/src/lib/select/select.spec.ts [hunk 1]
```diff
@@ -537,4 +538,34 @@ describe('vwc-select', () => {
...
+			element = (await fixture(
+				`<${COMPONENT_TAG}>
+					<option value="1">1</option>
+					<option value="2" selected>2</option>
+					<option value="3">3</option>
+				</${COMPONENT_TAG}>`
+			)) as Select;
+
+			element.placeholder = 'placeholder';
+			await elementUpdated(element);
```
Comment (RichardHelm) [inline libs/components/src/lib/select/select.spec.ts:566]: Why does this test not have the placeholder here?

It doesn't seem to work when setting both the placeholder and `selected` / `value`

```suggestion
		it('should change selectedindex -1 when option selected', async () => {
			element = (await fixture(
				`<${COMPONENT_TAG} placeholder="placeholder">
					<option value="1">1</option>
					<option value="2" selected>2</option>
					<option value="3">3</option>
				</${COMPONENT_TAG}>`
			)) as Select;

			await elementUpdated(element);
```

File: libs/components/src/lib/select/select.spec.ts [hunk 2]
```diff
@@ -537,4 +538,34 @@ describe('vwc-select', () => {
 			expect(element.selectedOptions[0]).toEqual(element.querySelector('option:nth-child(2)'));
 		});
 	});
+
+	describe('placeholder', function () {
```
Comment (RichardHelm) [inline libs/components/src/lib/select/select.spec.ts:542]: Can we add some more tests here please?
Maybe that the placeholder is displayed and that, like selectedIndex, value will be as expected

File: libs/components/src/lib/select/select.spec.ts [hunk 3]
```diff
@@ -537,4 +538,34 @@ describe('vwc-select', () => {
...
+		it('should set selectedindex -1 when placeholder', async () => {
```
Comment (RichardHelm) [inline libs/components/src/lib/select/select.spec.ts:543]:
```suggestion
		it('should set selectedindex -1 when placeholder is set', async () => {
```

File: libs/components/src/lib/select/select.ts [hunk 4]
```diff
@@ -28,20 +28,55 @@ export type SelectShape = Extract<Shape, Shape.Rounded | Shape.Pill>;
...
+	 * The placeholder attribute.
 	 *
-	 * Slot observer:
+	 * @public
+	 * HTML Attribute: placeholder
+	 */
+	@attr placeholder: string | undefined;
+
+	/**
+	 * The ref to the internal `.control` element.
```
Comment (RichardHelm) [inline libs/components/src/lib/select/select.ts:67]:
```suggestion
	 * The ref to the internal placeholder option.
```

**Review by RichardHelm — CHANGES_REQUESTED** [review]

**Review by RichardHelm — COMMENTED** [review]
Looks good now!
Just the screenshots seem to be failing now

**Comment by rinaok** [comment]
> Looks good now! Just the screenshots seem to be failing now

@RichardHelm They didn't pass because of the date-picker.

**Review by RichardHelm — APPROVED** [review]

**Pull request merged**